scripts/since-latest-release.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
```

Comment:
This script is a useful tool for developers working on packages to quickly view changes since the last release, which can be helpful for changelog generation, code reviews, or general development tracking.